### DISCUSSION TITLE
How to Remove whitespaces from line chart

### DISCUSSION THREAD

**Discussion by spsec111** [carbon-design-system/carbon-charts#1478] [Q&A] [answered]
![myChart](https://user-images.githubusercontent.com/114723945/206958519-3db1a2d6-4ec8-4dff-a7ba-166a70964dd9.png)

I want to render a chart for say 24 month period but data exist for only part of the timeperiod. Is there any way to remove the whitespaces and to indicate no data

**Comment by theiliad** [comment, marked as answer]
Hi 👋🏻  unfortunately we currently do not offer a feature to mark up those areas as `no data`
